app/src/main/java/org/mozilla/fenix/components/Core.kt

```diff
@@ -58,7 +59,10 @@ class Core(private val context: Context) {
             suspendMediaWhenInactive = !FeatureFlags.mediaIntegration
         )
 
-        GeckoEngine(context, defaultSettings, GeckoProvider.getOrCreateRuntime(context))
+        val engine = GeckoEngine(context, defaultSettings, GeckoProvider.getOrCreateRuntime(context))
+        WebCompatFeature.install(engine)
+
+        engine
```

Comment:
nit or Kotlin FYI: This can also be written as:

```Kotlin
GeckoEngine(context, defaultSettings, GeckoProvider.getOrCreateRuntime(context)).also {
    WebCompatFeature.install(it)
}
```

Reply:
Oh, that's cool! Fixed.